### PR TITLE
Use babel plugin to prevent optional chaining in generated code

### PR DESCRIPTION
**Pull request by kunalmohan**
Follow up from #220 

Please do not merge this yet. I'll test this locally to make sure we don't need any more changes.
(Sorry! probably should have done this before opening this PR)

### CODE REVIEW

**Comment by kainino0x** [comment]
Looks great so far! Tested locally to make sure it has minimal impact on the generated code (and it does) :)

File: babel.config.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = function (api) {
     presets: ['@babel/preset-typescript'],
     plugins: [
       '@babel/plugin-proposal-class-properties',
+      '@babel/plugin-proposal-optional-chaining',
```
Comment (kainino0x) [inline babel.config.js:7]: Please add a comment about why we use this plugin even though it's supported in core.

**Comment by kunalmohan** [comment]
I have added the comment and tested it locally. I am able to run the tests now, so these changes are enough.

**Comment by kunalmohan** [comment]
@kainino0x how soon will these changes be reflected in the live tests and wpt?

**Comment by kainino0x** [comment]
Normally they are automatically pushed to the live tests, I thought I set it up correctly for the main branch rename but I guess not. Thanks for catching that, I'll check on it.

WPT is rolled manually (currently only done through Chromium) so it won't happen until next time we get to it. Do you need it?

**Comment by kunalmohan** [comment]
Updating the live tests should be enough at the moment.
I'll ping you once when we start using the wpt (if it hasn't been updated by then).
Thank you!

**Comment by kainino0x** [comment]
Sounds good!

Live version fixed and rolled out: https://gpuweb.github.io/cts/out/common/framework/version.js

**Comment by kunalmohan** [comment]
@kainino0x Now will be a good time to update cts in WPT :)

**Pull request merged**